packages/composables/package.json

```diff
@@ -17,12 +17,17 @@
 	"license": "GPL-3.0",
 	"author": "Rijk van Zanten <rijkvanzanten@me.com>",
 	"exports": {
-		".": "./dist/index.js",
+		".": {
+			"types": "./dist/index.d.ts",
+			"import": "./dist/index.js"
+		},
 		"./package.json": "./package.json"
 	},
 	"main": "dist/index.js",
+	"types": "dist/index.d.ts",
```

Comment:
That does indeed work, the default behavior is to look for a co-located `*.d.ts` file. 
I just wanted to be as explicit as possible, especially since I didn't find a consistent style among the packages, some declared types some didn't. 
But I'm happy to remove the "types" entrypoint again.